### DISCUSSION TITLE
Scoring DNA with lineages

### DISCUSSION THREAD

**Discussion by hmms117** [evo-design/evo#71] [Q&A]
Hi,

First of all, great work with Evo and for sharing the models. 

Since you can prompt the model with lineages, I am wondering if it is possible to calculate scores or log likelihoods for a DNA sequence given a specific lineage? For example, would a prompt in the following style work?

```|d__Bacteria;p__Firmicutes;c__Bacilli;o__Bacillales;f__Bacillaceae;g__Bacillus;s__Bacillus subtilis | ATGA..<DNA>```

Many thanks

**Comment by brianhie** [comment]
This could definitely be possible!

Note that there are no spaces separating the right vertical bar and the DNA
